python/iceberg/api/filterable.py

```diff
@@ -17,7 +17,7 @@
 
 
 class Filterable(object):
-    ALL_COLUMNS = ("*",)
+    ALL_COLUMNS = ["*"]
```

Comment:
This seems like an odd change to me. Why wouldn't we use an immutable value?

Reply:
We use tuples and list mixed together throughout the codebase:
https://github.com/apache/incubator-iceberg/blob/57b10995aade1362e582cef68b20014023556501/python/iceberg/core/manifest_reader.py#L36

I find a list more appropriate than a tuple for the columns.

Reply:
For constants, I'd prefer to use a value that can't be modified accidentally.